### DISCUSSION TITLE
Observed performance score

### DISCUSSION THREAD

**Discussion by SivasubramanianV** [GoogleChrome/lighthouse#16031] [Q&A]
In simulated throttling run, we get 
Observedfcp, observedlcp, observedsi, observedcls, 
Is there an option to get observedperformancescore?

**Comment by adamraine** [comment]
No, we only calculate the performance score for the reported metrics. If you're interested you can use the [score calculator](https://googlechrome.github.io/lighthouse/scorecalc/#FCP=1125&LCP=1231&TBT=0&CLS=0&SI=1125&TTI=1125&FMP=1125&device=mobile&version=11.7.0).

**Reply by SivasubramanianV** [reply]
@adamraine also i saw that we dont have observed TBT. Basically i dont want to run in simulated and provided conditions 2 times, so i thought i could take all provided values from observed when i run simulated.
